roles/openshift_control_plane/tasks/main.yml

```diff
@@ -256,7 +256,7 @@
   retries: 60
   delay: 5
   with_items:
-  - "{{ 'etcd' if inventory_hostname in groups['oo_etcd_to_config'] else omit }}"
+  - "{{ 'etcd' if (inventory_hostname in groups['oo_etcd_to_config'] and inventory_hsotname in groups['oo_masters_to_config']) else omit }}"
```

Comment:
There is a typo in second inventory_hostname (hsotname).

Additionally, this does not address the issue.

Need to add when condition:
```
when:
- item != ''
```
and set item to empty string instead of omit.

Reply:
Thanks, updated.